api/handlers.go

```diff
@@ -1433,3 +1729,31 @@ func diffBFs(reqIDs []int64, bfFromDB []*models.BookmarkFolder) ([]int64, []int6
 	}
 	return forAdd, forDel
 }
+
+func diffTags(reqUIDs []string, bfFromDB []*models.LabelTag) ([]string, []string) {
```

Comment:
I was try to reuse diff ints but if use interface will need add custom cast. tell if you have other solution